### ISSUE TITLE
No more update possible

### ISSUE THREAD

**Issue by Socolin** [SpecFlowOSS/SpecFlow.Rider#214]
Hello,

In short, I cannot update the plugin on the Jetbrains marketplace anymore.

I don't have the administrator access of the [Specflow organization](https://plugins.jetbrains.com/organizations/SpecFlow) on the jetbrains marketplace to configure the [trader status](https://plugins.jetbrains.com/docs/marketplace/trader-status.html#trader-and-non-trader-status) 

![image](https://github.com/SpecFlowOSS/SpecFlow.Rider/assets/539620/a5291325-0bc1-433b-a68a-0b1e0c270dfa)

Tricentis (the company behind) Specflow seems to have abandoned the project (I'll let you check more information here https://reqnroll.net/news/2024/02/from-specflow-to-reqnroll-why-and-how/). So there is nobody I can contact to switch this settings. As a result I'll continue the work with the Reqnroll plugin. I'll try to keep de compatibility with Specflow (As long as it does not requires too much works)

If you have any pending issue, I suggest that you re-open it on Reqnroll plugin repo.

**Comment by Socolin** [comment]
Yes, uninstall specflow plugin first to avoid conflict at startup

**Comment by nathanpovo** [comment]
Link to the repository for the reqnroll plugin https://github.com/reqnroll/Reqnroll.Rider
